gc/base/ObjectScanner.hpp

```diff
@@ -19,14 +19,20 @@
 #if !defined(OBJECTSCANNER_HPP_)
 #define OBJECTSCANNER_HPP_
 
+#include "omrcfg.h"
 #include "ModronAssertions.h"
 #include "objectdescription.h"
 
 #include "BaseVirtual.hpp"
-#include "EnvironmentStandard.hpp"
+#include "EnvironmentBase.hpp"
 #include "GCExtensionsBase.hpp"
 #include "SlotObject.hpp"
 
+/* TODO: define & configure OMR_GC_LEAF_BITS for omrcfg.h */
```

Comment:
Yes. My intention. Would also like to discuss renaming MM_MarkingDelegate functions (current names are kind of disjointed IMO).